src/core/style/expressions/aggregation.js

```diff
@@ -14,16 +14,22 @@ function genAggregationOp(aggName, aggType) {
     return class AggregationOperation extends Expression {
         constructor(property) {
             checkInstance(aggName, 'property', 0, Property, property);
-            super({ property: property });
+            super({ property });
             this._aggName = aggName;
             this.type = aggType;
         }
         get name() {
             return this.property.name;
         }
+        get aggName() {
```

Comment:
I don't think getters are a good idea...

Reply:
Disagree

Reply:
Don't you like the getter? I think we should avoid accessing underscored variables

Reply:
I preffer `getAggregationName`